render/render.js

```diff
@@ -800,21 +793,26 @@ module.exports = function() {
 			// Add new style properties
 			for (var key in style) {
 				var value = style[key]
-				if (value != null) element.style.setProperty(normalizeKey(key), String(value))
+				if (value != null) {
+					if (key.indexOf("-") > -1) element.style.setProperty(key, String(value))
```

Comment:
Minor nit, since we do assume an ES6 global environment:

```suggestion
					if (key.includes("-")) element.style.setProperty(key, String(value))
```

Reply:
OK, I fixed it.